docs/source/markdown/podman-manifest-rm.1.md.in

```diff
@@ -0,0 +1,31 @@
+% podman-manifest-rm 1
+
+## NAME
+podman\-manifest\-rm - Remove manifest list or image index from local storage
+
+## SYNOPSIS
+**podman manifest rm** *list-or-index* [...]
```

Comment:
Also must add `[*options*]` here

Reply:
Nice. the .in file should not have been checked in.